### DISCUSSION TITLE
What is `q_app = cast(QApplication, None)` in qutebrowser/app.py for?

### DISCUSSION THREAD

**Discussion by user202729** [qutebrowser/qutebrowser#6393] [Q&A] [answered]
Is this simply something that should have been removed, or does it serve some purpose?

As far as I can see, it's `None` in the whole program.

**Comment by The-Compiler** [comment, marked as answer]
Looks that was an oversight in c1758230200814e84055f3e2954cb69f0fddf5d8 - I removed it in 72facbb8f2d1304539e91e290c695e9f0441807e, thanks!
